.github/workflows/publish-docs.yml

```
@@ -0,0 +1,231 @@
+                      <a href="https://github.com/query4j/dynamicquerybuilder">GitHub Repository</a> |
+                      <a href="https://github.com/query4j/dynamicquerybuilder/issues">Report Issues</a> |
+                      <a href="https://github.com/query4j/dynamicquerybuilder/blob/master/CONTRIBUTING.md">Contributing</a>
+                  </p>
+                  <p style="margin-top: 10px;">Built with ❤️ by the Query4j team</p>
+              </div>
+          </body>
+          </html>
+          EOF
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)